src/bitgo/keyutil.js

```diff
@@ -0,0 +1,44 @@
+var ecurve = require('ecurve')
+var curve = ecurve.getCurveByName('secp256k1')
+const BigInteger = require('bigi')
+const ECPair = require('../ecpair')
+
+/**
+ * Create an ECPair from the raw private key bytes
+ * @param {Buffer} buffer - Private key for the ECPair. Must be exactly 32 bytes.
+ * @param {Object} [network] - Network for the ECPair. Defaults to bitcoin.
+ * @return {ECPair}
+ */
+function privateKeyBufferToECPair (buffer, network) {
+  if (!Buffer.isBuffer(buffer) || buffer.length !== 32) {
+    throw new Error('invalid private key buffer')
+  }
+
+  var d = BigInteger.fromBuffer(buffer)
+
+  if (d.signum() <= 0 || d.compareTo(curve.n) >= 0) {
+    throw new Error('private key out of range')
+  }
+
```

Comment:
This function seems to have gotten both moved from `ECPair.fromPrivateKeyBuffer`, renamed, and changed in regard to dropping the section

```
  if (!ecPair.__Q && curve) {
    ecPair.__Q = ecurve.Point.decodeFrom(curve, fastcurve.publicKeyCreate(d.toBuffer(32), false))
  }
```

Could you clarify why the section to calculate missing public keys is no longer needed here?

Reply:
The `__Q` property is computed on demand here: 

https://github.com/BitGo/bitgo-utxo-lib/blob/master/src/ecpair.js#L44-L52

The section I removed isn't really necessary.

On second glance the removal changes behavior a little bit, since it uses `fastcurve.publicKeyCreate`. However, it is used incorrectly since `fastcurve` funcs can return `undefined`, which wasn't handled here. (See https://bitgoinc.atlassian.net/browse/BLOCK-266)

I will modify the site where `__Q` is created on-demand and use the accelerated curve methods there to avoid unexpected slowdowns.